### DISCUSSION TITLE
什麼是真正的學習

### DISCUSSION THREAD

**Discussion by Teddy111802518** [1101-GS4719-C/Python-Community-GS4719-C#2119] [General]
大家覺得什麼是真正的學習呢?反正絕對不是背書考試，我認為學習的技能不只用於讀書考試，而是在遇上未知的時候，如何去想辦法解決問題，這才是學習這件事的本質所在

**Comment by Teddy111802518** [comment]
啊我個人是在高中的時候既有這樣的想法，但迫於升學壓力，我遇到不會或好奇的東西我沒有時間去探討，因為背書考都來不及了，現在能自由學習真好

**Comment by christaaah** [comment]
我認為學習是從外在環境中獲取自己不熟悉的知識，並將其內化成自己的修養的方法。所以我認為就算自己想要學習的知識以後用不到、或對生活沒幫助也可以去學習，因為它會成為自己的一部分。

**Reply by chia-kai-chang** [reply]
嗚嗚，大家都好棒~ 大家果然都是大學生~

**Comment by forestdew507** [comment]
同意無論如何都還是可以去學習！有時候甚至會驚喜的在某個時刻發現那個知識/技能派上用場了～
有點偏題的再分享，我個人覺得高中選修科目知識更包含了一些「專業」的程度在裡面，至於國中的部分（儘管很多人會說以後又不會有人問你什麼是有機物無機物之類的），雖然從教學節奏與升學目標導向來看真的頗填鴨式教學，我自己現在也很弱的把很多沒再持續接觸的社會科知識遺失了，但不得不說，有時遇到其他輸入（文章/人）的時候會覺得國中的填鴨好像讓我得到了一些 common sense。總之覺得就算會忘記但有餘力時多吸收總是好的:)

**Comment by yuan3x2** [comment]
任何新知的獲取都可算是學習，但我認為更重要的是如何運用所學，能夠消化並所學的知識往往是最難的，因此能否運用所學或許能夠當成成功學習新知的指標？

**Comment by CeBru13** [comment]
在有興趣的地方進行鑽研，並在遇到問題時嘗試解決或求教，在這過程中不斷的進步

**Comment by NiCL100** [comment]
我覺得不論是自發地或被迫地學習，只要有所吸收、獲得，都稱得上真正的學習。畢竟都是花費心思與精力在鑽研，倘若沒有付出，應該不會有所收穫。因此，我認為有所付出地研習一門學問或一項技藝，就是真正的學習。

**Comment by Mminniehuang** [comment]
我覺得學習這件事最重要的是自發性的思考。
這學期我選修了一門在探討社會學的課程，期望我們透過不斷的思考，發現每個問題的多面性。而這對一直以來接受填鴨式教育，只會被動性思考、回答標準答案的我來說非常困難，緊張、毫無想法、無法結合過往學習的任何事物。因此也讓我發覺自己在學習上的困境，我似乎失去了自發思考的能力，也根本沒有「真正的學習」。

**Comment by poyen0806** [comment]
我覺得學習是一種行動的過程，而此行動的結果是讓自己獲得經驗得到成長。

如果是有目的性地學習，應是由上而下(Top Down)去尋找自己該學什麼，像是如果我要學機器學習，那我就會想到我會需要上一些機器學習的課，但在這之前我得先學離散數學或是線性代數，選擇學習一種語言的機器學習框架，所以我從Python開始學，一層一層往上爬。

但是在大學裡通常是沒有目的的，我們都會先去學很多東西，再把我們學的合併往上尋找下一步要學什麼，像是我學了微積分跟Python，下一步我可能就會選擇資料處理，複習我所學的同時進一步學習更深的知識。

**Comment by 112121py** [comment]
我覺得學習沒有一定的形式，最常見的，我們可以在學校學習一些簡單基礎的知識，我們也可以因為參加活動或比賽學習到合作以及課外的東西。再者，無論你做了什麼大大小小的事情，不論失敗或成功也都是一種學習，失敗了，我們可以覆盤自己是那些地方沒有做好，成功了則要審視自己在哪個地方保持下去，這都算是一種學習。

**Comment by ShuliANN** [comment]
我覺得學習是不限領域、形式的，由內心對某種沒有探索過的事物，激發想去探索的好奇心和熱情，且我們不只是記住事實或解決問題，而是培養一種理解世界的方式，並透過不斷挑戰自我來提升個人能力。

**Comment by Jinyuu17** [comment]
真正的學習不僅僅是記住資訊或完成一項任務，而是能夠內化、理解並運用所學的知識。在這裡有幾個關鍵元素來定義什麼是「真正的學習」：

1. 理解：真正的學習不僅僅是背誦或複述，而是能夠透徹理解所學的概念。這意味著你能解釋一個概念，並且知道它為什麼重要或如何運作。
2. 應用：當學到的知識能夠在不同的情境中運用時，這才是真正的掌握。學習不僅限於理論，還包括如何應對現實世界的挑戰，並能靈活地將知識轉化為技能。
3. 批判性思維：真正的學習需要挑戰和反思現有知識。學習者不應該盲目接受資訊，而應該提出問題，進行比較和評估，並形成自己的見解。
4. 持續學習與反思：學習是個持續的過程，而不是一個終點。真正的學習者會定期反思自己的進步和理解，並持續探索新知識。
5. 動機和好奇心：內在的學習動機與好奇心是推動學習的重要因素。當你對一個主題有真正的興趣時，你的學習動機會更加強烈，並且更容易記住和理解相關知識。
6. 從錯誤中學習：錯誤是學習過程的一部分。真正的學習者會從錯誤中反思，並改進自己的理解與方法。

總結來說，真正的學習是深度理解、應用、批判性反思與持續探索的結合。這不僅有助於個人技能的發展，還能在生活和職業中取得更長遠的成功。

**Comment by lai1008** [comment]
我覺得真正的學習就是可以把讀進去的東西運用出來，像是學到一個物理公式就可以理解它然後在相對應的題目使用，或是學到微積分的時候可以在遇到困難問題的時候可以解出答案。

**Comment by DarsooK** [comment]
我認為真正的學習是掌握理解、思考和解決問題的能力，而不僅僅是死記硬背或應付考試。它包括以下幾個核心：

1.理解與內化：能真正理解知識的內涵，並將其內化為自己的觀念。
2.應用與實踐：將所學的技能和知識用於解決實際問題，而非僅停留在書本上。
3.探索與創新：面對未知時，有能力主動探索、分析並創造新解決方案。
4.持續成長：保持好奇心和學習的習慣，隨時準備應對變化和挑戰。

學習的本質在於不斷提升自己應對世界的能力，而不只是追求分數或標籤。

**Comment by 113607514** [comment]
獲取任何新知都可以算是學習，但我認為更重要的是如何將所學付諸實踐。能夠消化並靈活運用學到的知識，往往才是學習中最具挑戰的一環。因此，能否將所學應用於實際情境，或許可以作為成功掌握新知的一個重要指標。

**Comment by wcyeee** [comment]
我也覺得學習一定不是背書考試！我個人覺得學習是可以從自己的錯誤或者是現有的知識中把這些東西吸收成自己的，並且是真的可以運用出來的！不然只會考試根本也不叫真正的學習，因為大部分的人考完就忘記了...

**Comment by yue083139** [comment]
我個人認為，學習並不只限於書本及學業上的知識，只要是透過某些經驗的總結來讓自己的行為、人格有哪些成長都可以是屬於學習的一種，例如從辦營隊當中學習到跟小孩子互動的技巧、跟同儕工作時如何才能更有效的溝通等等，是會在不知不覺中內化的一種隱性的學習。

**Comment by Elaine4488** [comment]
我覺得有時候學習不是學會什麼技能，而是慢慢了解自己、也了解別人。像是跟朋友討論，或是自己靜下來想事情，其實都是學習的一部分，重點不在形式，而是那個過程。當你遇到跟自己原本認知完全相反的東西，能試著放下偏見去理解，讓思維更靈活。

**Comment by Elaine4488** [comment]
從錯誤中成長也是一種學習。每當我做錯選擇或遇到失敗，當下的確會覺得很沮喪，甚至有點想放棄。但過一段時間後，我發現這些經歷反而讓我變得更堅強。如果能從中學到一些經驗，找到更好的方法，這樣下次就能避免重蹈覆轍，那麼這次的錯誤對我來說就變得很有價值，因為它幫助我成長了。
